Documentation/doc/Documentation/advanced/Configuration_variables.txt

```diff
@@ -72,12 +72,12 @@ and will serverly limit performances.
 
 | Variable              | Description                                                           | Type        | %Default Value |
 | :-                    | :-                    | :-   |  :-            |
-| `CMAKE_BUILD_TYPE`    | Indicates type of build. Possible values are 'Debug' or 'Release'     | CMake       |  |
-| `CMAKE_INSTALL_PREFIX`| Installation directory path                                           | CMake       | Debug |
+| `CMAKE_BUILD_TYPE`    | Indicates type of build. Possible values are 'Debug' or 'Release'     | CMake       | Release |
+| `CMAKE_INSTALL_PREFIX`| Installation directory path                                           | CMake       | `/usr/local` |
 | `CMAKE_C_COMPILER`    | Full-path to the executable corresponding to the C compiler to use.   | CMake       | platform-dependent |
 | `CMAKE_CXX_COMPILER`  | Full-path to the executable corresponding to the C++ compiler to use. | CMake       | platform-dependent |
 | `CXX`                 | Idem                                                                  | Environment | Idem |
-| `BUILD_SHARED_LIBS`   | Whether to build shared or static libraries.                          | CMake       | TRUE |
+| `BUILD_SHARED_LIBS`   | Whether to build shared or static libraries.                          | CMake       | ON |
```

Comment:
- As far as I know, the default of `CMAKE_BUILD_TYPE` is actually empty. See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html.
- `CMAKE_INSTALL_PREFIX` has a different default on Unix and Windows. Anyway, you ar right is default *is not* `Debug`.
- As for `BUILD_SHARED_LIBS`. `ON` and `TRUE` are equivalent for CMake. Why do want to change `TRUE` to `ON`?

Reply:
* change it from `Release` to `none`
* change `/usr/local` to `platform-dependent`
* You're right, but most projects use `ON/OFF` to set `BUILD_SHARED_LIBS`, and the other commands in this file use `ON/OFF` too.

So, the sentence in this file also needs to be updated.
https://github.com/CGAL/cgal/blob/e1d4a96401e30cb2a3035e97c8bda44c15696cc9/Documentation/doc/Documentation/advanced/Installation.txt#L81

Reply:
> Thank you. Should I do the following?
> 
>     * change it from `Release` to `none`

Yes.

>     * change `/usr/local` to `platform-dependent`

That's appropriate.

>     * You're right, but most projects use `ON/OFF` to set `BUILD_SHARED_LIBS`, and the other commands in this file use `ON/OFF` too.

I understand. So let's switch to `ON` as you proposed in your patch.

> So, the sentence in this file also needs to be updated.
> 
> https://github.com/CGAL/cgal/blob/e1d4a96401e30cb2a3035e97c8bda44c15696cc9/Documentation/doc/Documentation/advanced/Installation.txt#L81

Oh, right! Thanks for pointing that out. Would you willing to propose a patch for that sentence as well?

Reply:
@lrineau Yes, I update the code.